vignettes/hover-label.Rmd

```diff
@@ -0,0 +1,167 @@
+---
+title: "Boxly Hover Label"
```

Comment:
Can we enrich the title something like "Customize hover labels in interactive box plots"?

Reply:
Updated